frontend/nextjs/pages/api/token.ts

```diff
@@ -1,7 +1,11 @@
 import { NextApiRequest, NextApiResponse } from 'next'
 import { PluggyClient } from 'pluggy-sdk'
 
-const { PLUGGY_CLIENT_ID, PLUGGY_CLIENT_SECRET } = process.env
+const { 
+  PLUGGY_CLIENT_ID,
+  PLUGGY_CLIENT_SECRET,
+  NEXT_PUBLIC_PLUGGY_UPDATE_ITEM_ID: ITEMID_TO_UPDATE 
+} = process.env
```

Comment:
Destructuring environment variables is not a good practice in `Nextjs`, and sometimes this brings errors.
(note: in some cases, it works, but is explicit in `Nextjs` docs to not do it)
In this case, the environment variables could be undefined. 
Docs: https://nextjs.org/docs/basic-features/environment-variables
And here is when they explain this: 
<img width="638" alt="Screen Shot 2022-01-25 at 14 19 37" src="https://user-images.githubusercontent.com/83733486/151026771-de6e39b0-ab84-40ee-84ac-2e49d78a065c.png">